data-model/src/main/avro/Event.avdl

```diff
@@ -34,6 +34,8 @@ protocol EventProtocol {
     // Event may be associated with multiple entities
     array<string> entity_id_list = [];
 
+    int resource_index = -1;
```

Comment:
This is not an list of indices?

Reply:
I initially modeled it that way, but a span should be associated with at most one resource.